atlassian/confluence.py

```diff
@@ -1737,6 +1737,29 @@ def get_group_members(self, group_name="confluence-users", start=0, limit=1000,
 
         return response.get("results")
 
+    def get_all_members(self, group_name="confluence-users", expand=None):
+        """
+        Get  collection of all users in the given group
+        :param group_name
+        :param expand: OPTIONAL: A comma separated list of properties to expand on the content. status
+        :return:
+        """
+        limit = 50
+        flag = True
+        step = 0
+        members = []
+        while flag:
+            values = self.get_group_members(group_name=group_name, start=limit * step, limit=limit, expand=expand)
```

Comment:
What if server limit is below limit?
```suggestion
            values = self.get_group_members(group_name=group_name, start=len(members), limit=limit, expand=expand)
```